invenio_circulation/views.py

```diff
@@ -206,3 +207,81 @@ def get(self, *args, **kwargs):
             )
 
         return jsonify({})
+
+
+def loan_update_blueprint(app):
+    """Create a blueprint for updating Loan."""
+    blueprint = Blueprint(
+        "invenio_circulation_update_loan", __name__, url_prefix=""
+    )
+
+    create_error_handlers(blueprint)
+    rec_serializers = {
+        "application/json": (
+            "invenio_records_rest.serializers" ":json_v1_response"
+        )
+    }
+    serializers = {
+        mime: obj_or_import_string(func)
+        for mime, func in rec_serializers.items()
+    }
+    loan_request = LoanUpdateResource.as_view(
+        LoanUpdateResource.view_name.format(CIRCULATION_LOAN_PID_TYPE),
+        serializers=serializers,
+        ctx={},
+    )
+
+    url = "circulation/loans/<{0}:pid_value>/update".format(LOANID_CONVERTER)
+    blueprint.add_url_rule(url, view_func=loan_request, methods=["POST"])
+    return blueprint
+
+
+def validate_update(data):
+    """Validate data before update loan."""
+    item_pid = data.get("item_pid")
+    item_exists = current_app.config.get("CIRCULATION_ITEM_EXISTS")(item_pid)
```

Comment:
`data.get("item_pid")` could return `None`, maybe better to check this first?

Reply:
For schema consistency I would say `item_pid = data.get("item_pid") or ""`